README.md

```diff
@@ -1,12 +1,40 @@
 # SDK test data
+
 ## Description
-This is a set of test cases bundled as UFC - universal flag configuration. The purpose of these test cases is to ensure SDK libraries are compliant with core application.
+
+This is a set of test cases bundled as UFC - universal flag configuration. The purpose of these test cases is to ensure
+SDK libraries are compliant with core application.
 
 ## Dependencies
+
 Node.JS v18, Jest v29, Typescript v4
 
 ## Usage
+
 1. install dependencies by runnning CLI command `yarn install`.
 2. Update content of the [flags-v1.json](ufc/flags-v1.json).
-3. Validate tests by runnning CLI command `yarn run validate:tests`. 
-4. Obfuscate file [flags-v1.json](ufc/flags-v1.json) by runnning CLI command `yarn run obfuscate:ufc`. It will update file [flags-v1-obfuscated.json](ufc/flags-v1-obfuscated.json) with obfuscated version of the file [flags-v1.json](ufc/flags-v1.json).
+3. Validate tests by runnning CLI command `yarn run validate:tests`.
+4. Obfuscate file [flags-v1.json](ufc/flags-v1.json) by runnning CLI command `yarn run obfuscate:ufc`. It will update
+   file [flags-v1-obfuscated.json](ufc/flags-v1-obfuscated.json) with obfuscated version of the
+   file [flags-v1.json](ufc/flags-v1.json).
+
+# SDK Package Testing
+
+[![Test Packaged SDKs](https://github.com/Eppo-exp/sdk-test-data/actions/workflows/test-sdk-packages.yml/badge.svg)](https://github.com/Eppo-exp/sdk-test-data/actions/workflows/test-sdk-packages.yml)  
+[![Test SDKs Remotely](https://github.com/Eppo-exp/sdk-test-data/actions/workflows/test-sdks-remote.yml/badge.svg)](https://github.com/Eppo-exp/sdk-test-data/actions/workflows/test-sdks-remote.yml)
+
+| SDK                                                                                     | Published Version                                                                                                                                                                                       | Test Status                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
the table looks beautiful when rendered; no one wants to see how the sausage is made